src/loaders/ObjectLoader.js

```diff
@@ -311,8 +311,10 @@ THREE.ObjectLoader.prototype = {
 			for ( var i = 0, l = json.length; i < l; i ++ ) {
 
 				var image = json[ i ];
+				var isAbsolute = SCHEME.test( image.url );
```

Comment:
Hmmm... What does `isAbsolute` mean? Also instead of adding `SCHEME` to the global scope I would just do this:

```javascript
var isAbsolute = /^(?:[a-z]+:)?(?://)?/.test( image.url );
```